app/models/account/billing.rb

```diff
@@ -87,6 +87,8 @@ def charge!(amount, options = {})
       gateway_options[:order_id]= options[:invoice].id
     end
 
+    gateway_options.reverse_merge!(payment_method_id: payment_detail.payment_method_id)
```

Comment:
Should this be Stripe-specific maybe? If we once remove Stripe, no one will remember why this is here.